test/e2e/webdriver/driver.js

```diff
@@ -1114,34 +1116,60 @@ class Driver {
   }
 
   async checkBrowserForConsoleErrors(_ignoredConsoleErrors) {
-    const ignoreAllErrors = _ignoredConsoleErrors.includes('ignore-all');
-
     const ignoredConsoleErrors = _ignoredConsoleErrors.concat([
       // Third-party Favicon 404s show up as errors
       'favicon.ico - Failed to load resource: the server responded with a status of 404',
       // Sentry rate limiting
       'Failed to load resource: the server responded with a status of 429',
       // 4Byte
       'Failed to load resource: the server responded with a status of 502 (Bad Gateway)',
+      // Sentry error that is not actually a problem
+      'Event fragment with id transaction-added-',
```

Comment:
q: should we remove this entry?

Reply:
thank you @HowardBraham , the issue hasn't been fixed yet, but since we are using Chrome 126, the issue won't appear often. However, I just saw one occurrence, so I agree it's actually good to have it until the issue is fixed :+1: 

https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/93876/workflows/5f26960a-937a-4709-8bcc-fe553dadb7e9/jobs/3495010/parallel-runs/14